SpatialGDK/Source/SpatialGDK/Public/SpatialView/ViewCoordinator.h

```diff
@@ -16,17 +17,30 @@ class ViewCoordinator
 
 	~ViewCoordinator();
 
-	// Moveable, not copyable.
+	// Non-copyable.
```

Comment:
Why change this? Is that what is written in other places?

Reply:
huh, I could have sworn it said non-moveable before- or at least that's why I remember changing it. Git has a different opinion though, so I'll defer to that, and I can put this back.

Reply:
done